{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/__init__.py

```diff
@@ -1 +1 @@
-
+default_app_config = 'app.apps.MainAppConfig'
```

Comment:
I think this needs to be `{{ cookiecutter.repo_name }}.app.apps.MainAppConfig` but confirmation would be nice.